telegram/ext/dispatcher.py

```diff
@@ -566,7 +573,8 @@ def add_error_handler(self, callback, run_async=False):
             See https://git.io/fxJuV for more info about switching to context based API.
         """
         if callback in self.error_handlers:
-            raise ValueError('The callback is already registered as error handler.')
+            self.logger.debug('The callback is already registered as error handler. Ignoring.')
```

Comment:
```suggestion
            self.logger.debug('The callback is already registered as an error handler. Ignoring.')
```

Or maybe: `Ignoring already registered error handler.` (I like this one better - shorter text)